lang/test/org/partiql/lang/eval/CompOptions.kt

```diff
@@ -39,9 +39,9 @@ enum class CompOptions(
         }
     ),
 
-    TYPED_OP_BEHAVIOR_HONOR_PARAMS(
```

Comment:
Previous usage of this enum haven't been deleted/edited in some kdoc. E.g. https://github.com/partiql/partiql-lang-kotlin/blob/6c9949c0ac26806b20cc5dd42619b81fa29b099a/lang/test/org/partiql/lang/eval/EvaluatingCompilerIsTests.kt#L15-L17

Reply:
Will update it